scripts/gulpfiles/chunks.json

```diff
@@ -253,7 +253,6 @@
     "./core/extensions.js",
     "./core/block.js",
     "./core/utils/string.js",
-    "./core/utils/object.js",
```

Comment:
what's up with this change? is this the last file that is using object.inherits?

Reply:
No clue, something removed it automatically. But if you expand this code block down, it looks like it still exists [link](https://github.com/google/blockly/pull/5947/files#diff-75165818332bb6d85e32df0a855c805ce6c0fe94790fd66067719077b9f2c810L262). So I don't think I need to add it back in?